src/host/_stream.cpp

```diff
@@ -368,6 +372,7 @@ void WriteCharsVT(SCREEN_INFORMATION& screenInfo, const std::wstring_view& str)
                 { "\x1b[?1004h\x1b[?9001h" }, // RIS: Focus Event Mode + Win32 Input Mode
                 { "\033[?1004h" } // DECSET_FOCUS: Focus Event Mode

```

Comment:
while you're here

```suggestion
                { "\x1b[?1004h" } // DECSET_FOCUS: Focus Event Mode
```

Reply:
Right, I can also fix the BEL vs ST thing too.